### PR TITLE
RND-40: Rename pom.xml properties, project packages and document take…

### PR DESCRIPTION
**Pull request by Nikitosique**
The following PR contains changes related to the initial stage of developing skeleton-based application.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,12 +8,12 @@
         <version>3.0.6</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
-    <groupId>com.intellias</groupId> <!-- change it -->
-    <artifactId>basic-sandbox</artifactId> <!-- change it -->
+    <groupId>com.devteam</groupId>
+    <artifactId>sandbox-based-app</artifactId>
```
Comment (nikitalebed) [inline pom.xml:12]:
```suggestion
    <artifactId>intellias-parking</artifactId>
```

File: pom.xml [hunk 2]
```diff
@@ -8,12 +8,12 @@
         <version>3.0.6</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
-    <groupId>com.intellias</groupId> <!-- change it -->
-    <artifactId>basic-sandbox</artifactId> <!-- change it -->
+    <groupId>com.devteam</groupId>
```
Comment (nikitalebed) [inline pom.xml:11]: let's not change the `groupId `

File: pom.xml [hunk 3]
```diff
@@ -8,12 +8,12 @@
...
+    <artifactId>sandbox-based-app</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <name>basic-sandbox</name> <!-- change it -->
-    <description>Basic service to start your own by copying and adjusting for your needs
-    </description> <!-- change it -->
+    <name>sandbox-based-app</name>
```
Comment (nikitalebed) [inline pom.xml:14]:
```suggestion
    <name>intellias-parking</name>
```

**Review by nikitalebed — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Nikitosique) [reply pom.xml]: Renamed.

File: pom.xml (hunk 2 above)
Comment (Nikitosique) [reply pom.xml]: Sure.

File: pom.xml (hunk 3 above)
Comment (Nikitosique) [reply pom.xml]: Renamed this one as well.

**Comment by Nikitosique** [comment]
Closing this PR as it isn't supposed to be merged to master.

**Pull request closed without merging**